packages/create-graphql/src/utils.js

```diff
@@ -40,3 +43,7 @@ export const verifyYeoman = async () => { // eslint-disable-line import/prefer-d
 
   return true;
 };
+
+export const verifyVersion = async () => { // eslint-disable-line import/prefer-default-export
+  return await updateNotifier({pkg}).notify();
```

Comment:
That way the notification will only work for the `create-graphql` package, this repo contains a Lerna project with both `create-graphql` (the CLI) and `generator-graphql` (the Yeoman generator that scaffolds the code), it's better to verify both of them since they can all be updated individually.

I also happened to run into an issue by trying to use `update-notifier` asynchronously, so I think it's better to use the callback function: https://github.com/lucasbento/reminders-cli/blob/master/src/utils/updateNotifier.js#L7-L33.